src/components/shared/card.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { ReactElement } from 'react';
+import { RelatedItem } from '@guardian/apps-rendering-api-models/relatedItem';
+import { css, SerializedStyles } from '@emotion/core';
+import { headline, textSans } from '@guardian/src-foundations/typography';
+import { remSpace } from '@guardian/src-foundations';
+import { Option, withDefault, map, fromNullable } from '@guardian/types/option';
+import { makeRelativeDate } from 'date';
+import { pipe2 } from 'lib';
+import { text, neutral } from '@guardian/src-foundations/palette';
+import BodyImage from 'components/bodyImage';
+import { Pillar, Design, Display } from '@guardian/types/Format';
+import { Image } from 'image';
+import { darkModeCss } from 'styles';
+import { RelatedItemType } from '@guardian/apps-rendering-api-models/relatedItemType';
+import { Pillar as contentPillar } from '@guardian/content-api-models/v1/pillar';
+import { getPillarStyles, pillarFromString } from 'pillarStyles';
+
+
+
+interface Props {
+    relatedItem: RelatedItem;
+    image: Option<Image>;
+}
+
+const listStyles = css`
+    background: white;
+    margin-right: ${remSpace[3]};
+    flex: 0 0 15rem;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    img {
+        width: 100%;
+        height: 100%;
+        position: absolute;
+        top: 0;
+        left: 0;
+    }
+
+    time {
+        ${textSans.small()};
+        color: ${text.supporting};
+        text-align: right;
+        width: calc(100% - ${remSpace[2]});
+        display: inline-block;
+    }
+`
+
+const anchorStyles = css`
+    color: ${neutral[7]};
+    text-decoration: none;
+    ${darkModeCss`
+        color: ${neutral[86]};
+        background: ${neutral[20]};
+    `}
+`;
+
+const headingStyles = css`
+    ${headline.xxxsmall()};
+    margin: 0;
+    padding: ${remSpace[2]};
+    min-height: 150px;
+`;
+
+const imageWrapperStyles = css`
+    padding-bottom: 56.25%;
+    position: relative;
+`;
+
+const relativeFirstPublished = (date: Option<Date>): JSX.Element | null => pipe2(
+    date,
+    map(date => <time>{makeRelativeDate(date)}</time>),
+    withDefault<JSX.Element | null>(null),
+);
+
+const cardStyles = (type: RelatedItemType, pillar: contentPillar): SerializedStyles => {
```

Comment:
Doesn't seem to have caused any problems here, but is there a concern over using a TS reserved word (`type`) for a variable name?